util/chplenv/utils.py

```diff
@@ -23,6 +23,13 @@ def get_chpl_home():
     return chpl_home
 
 @memoize
+def using_chapel_module():
```

Comment:
@gbtitus I should have made the differences more obvious. `os.environ.get('CHPL_HOME', '')` will set the default value of chpl_home to `''` if it's not in the env. If you do `os.environ.get('CHPL_HOME')` the default value will be `None`

Right now chpl_home won't ever be `None`, so this function will mistakenly return true if for example neither chpl_home or chpl_module_home are set.

This should be 
```python
def using_chapel_module():
    chpl_home = os.environ.get('CHPL_HOME')
    if chpl_home it not None:
        return chpl_home == os.environ.get('CHPL_MODULE_HOME')
    return False
```

Where the only difference between what you have no and this is that there is no second arg `''` to `os.environ.get`